examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -254,6 +266,28 @@ void DeviceCallbacks::OnColorControlAttributeChangeCallback(EndpointId endpointI
 }
 #endif
 
+void DeviceCallbacks::OnIdentifyPostAttributeChangeCallback(EndpointId endpointId, AttributeId attributeId, uint16_t size,
+                                                            uint8_t * value)
+{
+    if (attributeId == Clusters::Identify::Attributes::IdentifyTime::Id)
+    {
+        uint16_t identifyTime;
+        size == 2 ? identifyTime = (value[1] << 8) + value[0] : identifyTime = value[0];
```

Comment:
No, if `size != 2` then this is some sort of broken call, and you can't assume anything about the value....

Furthermore, if `size == 2`, then `value` points to a native-endian `uint16_t`, whereas this code is assuming little-endian.

In other words, once we check size == 2, the next step is:
```
    uint16_t identifyTime;
    memcpy(&identifyTime, value, size);
```